config/locales/de.yml

```diff
@@ -430,7 +430,7 @@ de:
       long: ! '%e. %B %Y'
       short: ! '%e. %b'
     month_names:
-    - 
+    -
     - Januar
```

Comment:
Can you remove these changes? The files are generated by localeapp. It will always insert the trailing space again on next update.